device-types/APC/SMT1000RMI2UC.yaml

```diff
@@ -0,0 +1,28 @@
+---
+manufacturer: APC
+model: Smart-UPS 1000VA, Rack Mount, LCD 230V with SmartConnect Port
```

Comment:
This is not the correct model name, realistically this is too long to be correct. It looks like `SMT1000RMI2UC` would be a better option

Reply:
@sthierolf please check this comment as well